tsl/src/bgw_policy/job.c

```diff
@@ -508,6 +512,12 @@ policy_refresh_cagg_read_and_validate_config(Jsonb *config, PolicyContinuousAggD
 
 	refresh_newest_first = policy_refresh_cagg_get_refresh_newest_first(config);
 
+	bool process_hypertable_invalidations_isnull;
+	bool process_hypertable_invalidations =
+		ts_jsonb_get_bool_field(config,
+								POL_REFRESH_CONF_KEY_PROCESS_HYPERTABLE_INVALIDATIONS,
+								&process_hypertable_invalidations_isnull);
+
```

Comment:
An open question here is if we when validating the options should check for unrecognized options.

- If the user makes a typo, it might go unnoticed and lead to hard to find configuration errors. Checking for unrecognized options will then make this apparent.
- If we do not allow unrecognized options, there is no way for the user to add their own options used for other purposes, and it makes version handling more complicated.

A good middle ground could be to print a warning about unrecognized options.